### PR TITLE
[IMP] accounting: default taxes in taxes.rst

### PR DESCRIPTION
**Pull request by gawa-odoo**
Since https://github.com/odoo/odoo/pull/115222, a new line on an invoice won't have the default sales tax put on by default. If there are no taxes on the product, the account or from predictive, the tax will stay empty Its use has been restricted back to the tax put on new products. It is still used in Accounting Firms mode, as we don't want people to have to input the product when quick encoding.

task-3045026

### CODE REVIEW

File: content/applications/finance/accounting/taxation/taxes/default_taxes.rst [hunk 1]
```diff
@@ -2,11 +2,10 @@
...
 =============
 
-**Default Taxes** define which :doc:`taxes <taxes>` are automatically selected when there is no
-other indication about which tax to use. For example, Odoo prefills the **Taxes** field with the
-Default Taxes when you create a new product or add a new line on an invoice.
+**Default Taxes** define which :doc:`taxes <taxes>` are automatically selected when creating a new product.
+It is also used to prefill the **Taxes** field with when you add a new line on an invoice in Accounting Firms mode.
 
-.. image:: default_taxes/default-taxes-invoice-line.png
+.. image:: default_taxes/default-taxes-product.png
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/taxation/taxes/default_taxes.rst:8]: For a better resolution, please take the screenshot with 110% zoom (or more if needed). Also, the file should always be compressed, if there are any irrelevant parts, it's better to crop them ( for example, here the internal note field at the bottom can be left out so we can focus on what matters).

File: content/applications/finance/accounting/taxation/taxes/default_taxes.rst [hunk 2]
```diff
@@ -2,11 +2,10 @@
 Default taxes
 =============
 
-**Default Taxes** define which :doc:`taxes <taxes>` are automatically selected when there is no
-other indication about which tax to use. For example, Odoo prefills the **Taxes** field with the
-Default Taxes when you create a new product or add a new line on an invoice.
+**Default Taxes** define which :doc:`taxes <taxes>` are automatically selected when creating a new product.
+It is also used to prefill the **Taxes** field with when you add a new line on an invoice in Accounting Firms mode.
```
Comment (chiaraprattico) [inline content/applications/finance/accounting/taxation/taxes/default_taxes.rst:6]: **Default Taxes** define which :doc:`taxes <taxes>` are automatically selected when creating a new product.
They also used to prefill the :guilabel:`Taxes` field when adding a new line on an invoice in Accounting Firms mode.

**Review by chiaraprattico — CHANGES_REQUESTED** [review]
Hello, thanks for this one. I made a few suggestions. You can find more information about adding images to a doc over here https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#images

**Review by chiaraprattico — APPROVED** [review]

**Comment by jcs-odoo** [comment]
oh no, this is going one of the many conflicts with the restructure :'(

**Comment by gawa-odoo** [comment]
> oh no, this is going one of the many conflicts with the restructure :'(

I don't know what the restructure is about, but I don't mind waiting to rebase and merge it afterwards. I mainly wanted to do it to not forget about this change.

**Comment by jcs-odoo** [comment]
> > oh no, this is going one of the many conflicts with the restructure :'(
> 
> I don't know what the restructure is about, but I don't mind waiting to rebase and merge it afterwards. I mainly wanted to do it to not forget about this change.

Haha! No, no worries @gawa-odoo go ahead and merge this whenever needed!

Here is our attempt at improving the structure of the accounting documentation.
https://github.com/odoo/documentation/pull/4422

**Review by jcs-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**